app/src/main/java/hu/tvarga/sunnyeats/app/SunnyEatsApplication.java

```diff
@@ -0,0 +1,54 @@
+package hu.tvarga.sunnyeats.app;
+
+import android.app.Activity;
+import android.app.Application;
+import android.app.Service;
+import android.support.annotation.NonNull;
+
+import javax.inject.Inject;
+
+import dagger.android.AndroidInjector;
+import dagger.android.DispatchingAndroidInjector;
+import dagger.android.HasActivityInjector;
+import dagger.android.HasServiceInjector;
+import hu.tvarga.sunnyeats.BuildConfig;
+import hu.tvarga.sunnyeats.app.di.di.ApplicationComponent;
+import hu.tvarga.sunnyeats.app.di.di.BaseApplicationModule;
+import hu.tvarga.sunnyeats.app.di.di.DaggerApplicationComponent;
+import timber.log.Timber;
+
+public class SunnyEatsApplication extends Application
+		implements HasActivityInjector, HasServiceInjector {
+
+	@Inject
+	DispatchingAndroidInjector<Activity> dispatchingActivityInjector;
+
+	@Inject
+	DispatchingAndroidInjector<Service> dispatchingServiceInjector;
+
+	@Override
+	public void onCreate() {
+		super.onCreate();
+		if (BuildConfig.DEBUG) {
+			Timber.plant(new Timber.DebugTree());
+		}
+		createComponent().inject(this);
+	}
+
+	@Override
+	public AndroidInjector<Activity> activityInjector() {
+		return dispatchingActivityInjector;
+	}
+
+	@NonNull
+	protected ApplicationComponent createComponent() {
+		return DaggerApplicationComponent.builder().baseApplicationModule(
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this use of "baseApplicationModule"; it is deprecated. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3ACallToDeprecatedMethod)